x-pack/elastic-agent/spec/metricbeat.yml

```diff
@@ -1,6 +1,13 @@
 name: Metricbeat
 cmd: metricbeat
-args: ["-E", "setup.ilm.enabled=false", "-E", "setup.template.enabled=false", "-E", "management.mode=x-pack-fleet", "-E", "management.enabled=true", "-E", "logging.level=debug"]
+args: [
+  "-E", "setup.ilm.enabled=false",
+  "-E", "setup.template.enabled=false",
+  "-E", "management.mode=x-pack-fleet",
+  "-E", "management.enabled=true",
+  "-E", "logging.level=debug",
+  "-E", "gc_percent=${METRICBEAT_GOGC:100}"
```

Comment:
FB / MB are enough for now, but I assume we will need to apply it to every Golang process / beat we run?

Reply:
Yes, if we find this works well, we will extend on this and document the env vars.